test/api/v3/unit/middlewares/cronMiddleware.js

```diff
@@ -107,53 +115,57 @@ describe('cron middleware', () => {
     await task.save();
     await user.save();
 
-    cronMiddleware(req, res, (err) => {
+    cronMiddleware(req, res, () => {
       Tasks.Task.findOne({_id: task}, function (secondErr, taskFound) {
         expect(secondErr).to.not.exist;
         expect(taskFound).to.not.exist;
-        done(err);
       });
     });
   });
 
-  it('should call next if user was not modified after cron', async (done) => {
+  it('should call next if user was not modified after cron', async () => {
+    execStub.returns(Bluebird.resolve(null));
+
     let hpBefore = user.stats.hp;
     user.lastCron = moment(new Date()).subtract({days: 2});
     await user.save();
 
-    cronMiddleware(req, res, (err) => {
+    cronMiddleware(req, res, () => {
```

Comment:
If I wasn't clear in my previous comment, if you add a `throw new Error()` here https://github.com/HabitRPG/habitica/blob/develop/website/server/middlewares/cron.js#L21 is it catched in these tests? Because you've removed error handling from the callbacj